.travis.yml

```diff
@@ -87,7 +89,7 @@ jobs:
       arch: arm64
       env: >-
         FILE_ENV="./ci/test/00_setup_env_arm.sh"
-        QEMU_USER_CMD=""  # Can run the tests natively without qemu
+        QEMU_USER_CMD=""
```

Comment:
Would it work to place the comment in an extra line instead, between `FILE_ENV=...` and `QEMU_USER_CMD=...`? The comment itself would still be helpful for anyone trying to understand or edit the Travis file IMHO.

Reply:
No, it wouldn't: the `>` character starts the "Folded Block Scalar". The closest line to place a comment is above `env: >-`.